.cookiecutter.yaml

```diff
@@ -5,6 +5,6 @@ default_context:
     project_name: "Speakerbox"
     project_slug: "speakerbox"
     python_slug: "speakerbox"
-    project_short_description: "Speaker Annotation using Audio Classification"
+    project_short_description: "A Workflow for Annotation and Training for Multi-Speaker Classification Models"
```

Comment:
I would suggest moving away from "workflow" in the title - JOSS usually does not publish a workflow , unless it is a workflow engine - see examples --> (https://joss.theoj.org/papers/search?q=Workflow)